src/Maestro/Maestro.Data/Models/BuildIncoherence.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Maestro.Data.Models
+{
+    public class IncoherentDependency
+    {
+        public string Name { get; set; }
+        public string Version { get; set; }
+        public string Repository { get; set; }
+        public string Commit { get; set; }
+    }
+
+    public class IncoherentNode
```

Comment:
What's the difference between an incoherent node and an incoherent dependency?

Reply:
Talked offline. I'll remove the nodes list.